tmt/steps/prepare/install.py

```diff
@@ -232,4 +232,4 @@ def go(self, guest):
             # Check and install (extra check for yum to workaround BZ#1920176)
             guest.execute(
                 f'{check} || {command} install {options} {packages}' +
-                (f' && {check}' if 'yum' in command else ''))
+                (f' && {check}' if 'yum' in command and not skip else ''))
```

Comment:
So do I understand correctly that `skip` means _recommend_ and the only issue with _yum_ is that it is sometimes giving _false positive_ but never _false negative_?

Reply:
Implementation updated to cover `yum install --skip-broken weird` returning a non-zero.